model/src/main/java/oracle/kubernetes/weblogic/domain/v2/BaseConfiguration.java

```diff
@@ -112,37 +75,13 @@ private boolean isStartNever() {
     return Objects.equals(getServerStartPolicy(), ConfigurationConstants.START_NEVER);
   }
 
-  private List<V1EnvVar> getV1EnvVars(BaseConfiguration configuration) {
-    return Optional.ofNullable(configuration.getEnv()).orElse(emptyList());
-  }
-
-  private void addIfMissing(V1EnvVar var) {
-    if (!hasEnvVar(var.getName())) addEnvVar(var);
-  }
-
-  private boolean hasEnvVar(String name) {
-    if (env == null) return false;
-    for (V1EnvVar var : env) {
-      if (var.getName().equals(name)) return true;
-    }
-    return false;
-  }
-
-  private void copyValues(V1Probe toProbe, V1Probe fromProbe) {
```

Comment:
Oh, I see that this is what you've implemented -- just copying the values.  Still, I don't think this will be understood by the customer.

Reply:
So, you'd prefer that we wrote our own liveness probe and readiness probe models, which are subsets of the k8s ones?  I could update our wiki to match (currently the wiki says we'll use the k8s ones and only pay attention to a subset of their properties).

Reply:
Yes, I didn't read the wiki well here.  It's amazing how many times you can read the same thing and see what you think you see rather than what is there.

I've looked at V1Probe and there are far too many fields, so this will be confusing.  Also, it implies that the customer can replace the probes fully, which we don't want to support.

So, this should be a collection of just the tuning fields.